multiqc/modules/bcftools/stats.py

```diff
@@ -112,45 +128,56 @@ def parse_bcftools_stats(self):
                     'anchor': 'bcftools-stats',
                     'content': linegraph.plot(self.bcftools_stats_indels, pconfig)
                 })
+            # Make line graph of variants per depth
+            if len(depth_data) > 0:
+                import pprint
+                pprint.pprint(depth_data)
+                pconfig = {'id': 'bcftools_stats_depth', 'title': 'Variant depths',
+                           'ylab': 'Fraction of sites (%)', 'xlab': 'Variant depth',
+                           'ymin': 0, 'ymax': 100, 'categories': True}
+                self.sections.append({'name': 'Variant depths',
+                                      'anchor': 'bcftools-stats',
+                                      'content': linegraph.plot(depth_data, pconfig)})
 
         # Return the number of logs that were found
         return len(self.bcftools_stats)
 
     def bcftools_stats_genstats_table(self):
         """ Add key statistics to the General Stats table """
         stats_headers = OrderedDict()
-        stats_headers['number_of_SNPs'] = {
-            'title': 'M SNPs',
-            'description': 'Number of SNPs (millions)',
-            'min': 0,
-            'format': '{:.2f}',
-            'modify': lambda x: x / 1000000
+        stats_headers['number_of_records'] = {
```

Comment:
Generally, I try to keep numbers in the General Stats under 1000 - hence the previous columns had the `'modify': lambda x: x / 1000000` to display the figures in millions.

My reasoning is that it's pretty difficult to spot the difference between 5522770 and 552277 when scanning a report quickly. The difference between 5.5 and 0.5 is easy though.